.drone.yml

```
@@ -0,0 +1,120 @@
+---
+kind: pipeline
+name: default
```

You might need to specify this explicitly, I don't remember
```suggestion
kind: pipeline
type: kubernetes
name: default
```